pkg/acl/access_control_list.go

```diff
@@ -8,34 +8,57 @@ import (
 )
 
 type DeniedUsers []string
+type AcceptedUsers []string
 
 func (deniedAccounts DeniedUsers) IsUserDenied(username string) bool {
 	return arrays.FindFirstString(deniedAccounts, func(user string) bool {
 		return username == user
 	}) != -1
 }
 
+func (acceptedAccounts AcceptedUsers) IsUserAccepted(username string) bool {
```

Comment:
same comment as https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1086/files#r893781287 for the method name and parameter